Makefile

```diff
@@ -190,6 +204,26 @@ nxMySQL:
 	mkdir -p ../../release; \
 	mv $@_$${VERSION}.zip ../../release/
 
+nxOMSAgent:
+	rm -rf output/staging; \
+	VERSION="1.0"; \
+	PROVIDERS="nxOMSAgent nxOMSCustomLog nxOMSPlugin nxOMSSyslog nxService nxPackage"; \
```

Comment:
Could add the following section for the nxOMSPlugin module?

nxOMSPlugin:
       rm -rf output/staging; \
       VERSION="1.0"; \
       PROVIDERS="nxOMSPlugin"; \
       STAGINGDIR="output/staging/$@/DSCResources"; \
       for current in $$PROVIDERS; do \
               mkdir -p $$STAGINGDIR/MSFT_$${current}Resource/$(PF_ARCH)/Scripts/{2.4x-2.5x,2.6x-2.7x,3.x}/Scripts; \
               cp Providers/Modules/$@.psd1 output/staging/$@/; \
        cp -r Providers/Modules/Plugins/ $$STAGINGDIR/MSFT_$${current}Resource/; \
               cp Providers/$${current}/MSFT_$${current}Resource.schema.mof $$STAGINGDIR/MSFT_$${current}Resource/; \
               cp Providers/$${current}/MSFT_$${current}Resource.reg $$STAGINGDIR/MSFT_$${current}Resource/; \
               cp Providers/bin/libMSFT_$${current}Resource.so $$STAGINGDIR/MSFT_$${current}Resource/$(PF_ARCH); \
               cp Providers/Scripts/2.4x-2.5x/Scripts/$${current}.py $$STAGINGDIR/MSFT_$${current}Resource/$(PF_ARCH)/Scripts/2.4x-2.5x/Scripts; \
               cp Providers/Scripts/2.6x-2.7x/Scripts/$${current}.py $$STAGINGDIR/MSFT_$${current}Resource/$(PF_ARCH)/Scripts/2.6x-2.7x/Scripts; \
               cp Providers/Scripts/3.x/Scripts/$${current}.py $$STAGINGDIR/MSFT_$${current}Resource/$(PF_ARCH)/Scripts/3.x/Scripts; \
       done;\
       cd output/staging; \
       zip -r $@_$${VERSION}.zip $@; \
       mkdir -p ../../release; \
       mv $@_$${VERSION}.zip ../../release/